helios-tools/src/main/java/com/spotify/helios/cli/command/JobHistoryCommand.java

```diff
@@ -90,7 +90,7 @@ int run(final Namespace options, final HeliosClient client, final PrintStream ou
 
     for (final TaskStatusEvent event : events) {
       final String host = checkNotNull(event.getHost());
-      final long timestamp = checkNotNull(event.getTimestamp());
+      final long timestamp = event.getTimestamp();
```

Comment:
same reasoning as above, `getTimestamp()` is a primitive value and not an object